openedx/features/custom_fields/multiselect_with_other/db/fields.py

```diff
@@ -93,8 +93,9 @@ def to_python(self, value):
             if isinstance(value, list):
                 return value
             elif isinstance(value, string_type):
-                choices_str = value.replace(self.other_delimiter, '')
-                selected_choices = [choice for choice in choices_str.split(',') if choice.strip()]
```

Comment:
Name choices_str is not serving the purpose here, can we please rename it?

Reply:
done